android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/EventHandler.kt

```diff
@@ -3,7 +3,7 @@ package com.devcycle.sdk.android.eventsource
 import timber.log.Timber
 
 class Handler(
-    private var onMessageHandler: () -> Any
+    private var onMessageHandler: (event: String?, messageEvent: MessageEvent?) -> Any
```

Comment:
will the handler ever care about both of these properties, or does it just want the messageEvent?

Reply:
good point, the event seems to always be 'message' so i can't see it ever being useful